alcs-frontend/src/app/shared/timeline/timeline.component.html

```diff
@@ -16,10 +16,10 @@
             class="dot"
           ></div>
         </td>
-        <td>{{ event.startDate | date: dateFormat }}</td>
+        <td>{{ event.startDate | momentFormat: dateFormat }}</td>
         <td>
           <span *ngIf="event.isFulfilled">{{
-            event.fulfilledDate ? (event.fulfilledDate | date: dateFormat) : '-'
+            event.fulfilledDate ? (event.fulfilledDate | momentFormat: dateFormat) : '-'
```

Comment:
Look at this face 😄. Obviously not in scope of this ticket, but I think it would be nice to have a pipeline for showing empty data

Reply:
Done! Pipe now defaults to - if no value was provided